README.md

```
@@ -1,39 +1,468 @@
+   ```
+
+2. **Install dependencies**
+   ```bash
+   npm install
+   # or with bun
+   bun install
+   ```
+
+3. **Set up environment variables**
+   Create `.env.local`:
+   ```env
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)